packages/devextreme/js/__internal/ui/m_tag_box.ts

```diff
@@ -1110,6 +1118,7 @@ const TagBox = (SelectBox as any).inherit({
     }
 
     let $tag = this._getTag(value);
+
```

Comment:
Seems this change is unnecessary

Reply:
[aea94cd](https://github.com/DevExpress/DevExtreme/pull/27157/commits/aea94cd1437771c23e23b2de7e712ee63a6db785)

Reply:
reverted my changes